### DISCUSSION TITLE
Gathering clean snapshots for Frigate+

### DISCUSSION THREAD

**Discussion by MrAnonymous722** [blakeblackshear/frigate#2960] [Ask A Question] [answered]
I can't seem to figure out an easy way to take a bunch of my past snapshots, make them clean and import them into images in Frigate+. Would I just need to manually specify `bounding_box: False` and take snapshots from that point forward, or is there a clean copy saved somewhere for past events that have bounding boxes? Would the thumbnails be acceptable for Frigate+ images?

**Comment by blakeblackshear** [comment, marked as answer]
In your clips directory, there is a version of every snapshot with `clean.png` as a suffix. Next version of Frigate will have a "Submit to Frigate+" button to make this easier.

**Reply by MrAnonymous722** [reply]
Excellent, thank you!
